### PR TITLE
sandbox tweak

### PR DESCRIPTION
**Pull request by xu-cheng**

### CODE REVIEW

File: Library/Homebrew/sandbox.rb [hunk 1]
```diff
@@ -8,11 +8,12 @@ class Sandbox
   ].freeze
 
   def self.available?
-    OS.mac? && File.executable?(SANDBOX_EXEC)
+    OS.mac? && OS::Mac.version >= "10.6" && File.executable?(SANDBOX_EXEC)
```
Comment (MikeMcQuaid) [inline Library/Homebrew/sandbox.rb]: Does that file exist and is executable on e.g. 10.5 but just doesn't work? If not I think we can leave it as-is.

File: Library/Homebrew/sandbox.rb [hunk 2]
```diff
@@ -8,11 +8,12 @@ class Sandbox
...
   end
 
   def self.formula?(formula)
     return false unless available?
+    return false if ARGV.no_sandbox?
```
Comment (MikeMcQuaid) [inline Library/Homebrew/sandbox.rb]: 👍 to this.

File: Library/Homebrew/sandbox.rb (hunk 1 above)
Comment (xu-cheng) [inline Library/Homebrew/sandbox.rb]: You can see commit message for explanation. Basically it exists but too old to support DSL we use.

Comment (MikeMcQuaid) [inline Library/Homebrew/sandbox.rb]: Ok 🆒 👍

**Pull request closed without merging**